Formula/git.rb

```diff
@@ -58,15 +64,16 @@ def install
     ENV["LIBPCREDIR"] = Formula["pcre2"].opt_prefix
     ENV["V"] = "1" # build verbosely
 
-    perl_version = Utils.safe_popen_read("perl", "--version")[/v(\d+\.\d+)(?:\.\d+)?/, 1]
-
     on_macos do
+      perl_version, perl_short_version = Utils.safe_popen_read("perl", "-e", "print $^V")
+                                              .match(/v((\d+\.\d+)(?:\.\d+))?/).captures
       ENV["PERLLIB_EXTRA"] = %W[
-        #{MacOS.active_developer_dir}
-        /Library/Developer/CommandLineTools
-        /Applications/Xcode.app/Contents/Developer
+        #{HOMEBREW_PREFIX}/lib/perl5/site_perl/#{perl_version}
```

Comment:
Shouldn't `perl_version` here be the version of Homebrew `perl`? Because `perl_version` will be the version of system `perl` on macOS given what you have here.

Reply:
No, `git` and `subversion` both build with System Perl, not Homebrew Perl